File: pa5/DictionaryTest.c
//DictionaryTest.c
//Cole Teza
#include<stdio.h>
#include<stdlib.h>
#include<string.h>
#include"Dictionary.h"

#define MAX_LEN 180

int main(int argc, char* argv[]){
   Dictionary A = newDictionary();
   char* k;
   char* v;
   char* word1[] = {"one","two","three","four","five","six","seven"};
   char* word2[] = {"foo","bar","blah","galumph","happy","sad","blue"};
   int i;

   for(i=0; i<7; i++){
      insert(A, word1[i], word2[i]);

   printDictionary(stdout, A);

   delete(A, "one");
   delete(A, "three");
   delete(A, "seven");

   printDictionary(stdout, A);

   printf("%s\n", (lookup(A));

   makeEmpty(A);
   printf("%s\n", (isEmpty(A)?"true":"false");

   freeDictionary(&A);

   return(EXIT_SUCCESS);
}

File: pa3/DictionaryTest.java
//DictionaryTest.java
//Cole Teza
//#1361038
//CMPS 12b PA3

public class DictionaryTest {

  public static void main(String[] args) {
    Dictionary A = new Dictionary();
	
      System.out.println(A.isEmpty());
	  
      A.insert("2","C");
      System.out.println(A.isEmpty());
	  
	  
      A.insert("1","L");

      System.out.println(A.size());
	  
      A.insert("6","E");
      A.insert("9","1");
      A.insert("5","3");

      System.out.println(A);
      System.out.println(A.size());
	  
      A.delete("6");
      System.out.println(A.size());
	  
      System.out.println(A);
	  
      System.out.println(A.lookup("5"));
	  
      A.makeEmpty();
      System.out.println(A.size() + " " + A.isEmpty());
  }
}